k8s/deployments/mariadb.yaml

```diff
@@ -33,26 +21,11 @@ spec:
                   name: mpm-secrets
                   key: DB_PASSWORD
             - name: MYSQL_DATABASE
-              value: micro_power_manager
+              value: database-1
```

Comment:
I think this has to be `micro_power_manager`. It has to be corresponding to what is configured in MPM connections.